modules/floating-buttons/assets/scss/frontend.scss

```diff
@@ -1,3 +1,5 @@
+@import "../../../../assets/dev/scss/helpers/variables";
```

Comment:
I would love to chat with Louis about this. Is there a way that we can load this at globally, so that we don't have to import it in each and every SCSS file?

@louiswol94